README.md

```diff
@@ -1,64 +1,31 @@
 linter-foodcritic
 =========================
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides an interface to [foodcritic](https://github.com/AtomLinter/linter-foodcritic). It will be used with files that have the “Ruby” syntax.
+This linter plugin for [Linter](https://github.com/atom-community/linter)
```

Comment:
I mean instead of using word `linter` twice it should've been used once IMHO. but I don't mind the way it is right now